yt_dlp/extractor/amazonminitv.py

```diff
@@ -0,0 +1,327 @@
+        'info_dict': {
+            'id': 'amzn1.dv.gti.0aa996eb-6a1b-4886-a342-387fbd2f1db0',
+        },
+    }, {
+        'url': 'amazonminitv:season:0aa996eb-6a1b-4886-a342-387fbd2f1db0',
+        'only_matching': True,
+    }]
+
+    def _entries(self, asin):
+        season_info = self._call_api(
+            asin,
+            data={
+                'operationName': 'getEpisodes',
+                'variables': {
+                    'episodeOrSeasonId': asin,
+                },
+                'query': '''
+query getEpisodes($sessionIdToken: String!, $clientId: String, $episodeOrSeasonId: ID!, $deviceLocale: String) {
+  getEpisodes(
+    applicationContextInput: {sessionIdToken: $sessionIdToken, deviceLocale: $deviceLocale, clientId: $clientId}
+    episodeOrSeasonId: $episodeOrSeasonId
+  ) {
+    episodes {
+      ... on Episode {
+        contentId
+        name
+        images
+        seriesName
+        seasonId
+        seriesId
+        seasonNumber
+        episodeNumber
+        description {
+          synopsis
+          contentLengthInSeconds
+        }
+        publicReleaseDateUTC
+      }
+    }
+  }
+}
+''',
+            },
+            note='Downloading season info')
+
+        for episode in season_info['episodes']:
+            yield self.url_result(f'amazonminitv:{episode["contentId"]}', AmazonMiniTVIE, episode['contentId'])
+
+    def _real_extract(self, url):
+        asin = f'amzn1.dv.gti.{self._match_id(url)}'
+        return self.playlist_result(self._entries(asin), playlist_id=asin)
+
+
+class AmazonMiniTVSeriesIE(AmazonMiniTVIE):
+    IE_NAME = 'amazonminitv:series'
+    _VALID_URL = r'amazonminitv:series:(?:amzn1\.dv\.gti\.)?(?P<id>[a-f0-9-]+)'
+    _TESTS = [{
+        'url': 'amazonminitv:series:amzn1.dv.gti.56521d46-b040-4fd5-872e-3e70476a04b0',
+        'playlist_mincount': 3,
+        'info_dict': {
+            'id': 'amzn1.dv.gti.56521d46-b040-4fd5-872e-3e70476a04b0',
+        },
+    }, {
+        'url': 'amazonminitv:series:56521d46-b040-4fd5-872e-3e70476a04b0',
+        'only_matching': True,
+    }]
+
+    def _entries(self, asin):
+        season_info = self._call_api(
+            asin,
+            data={
+                'operationName': 'getSeasons',
+                'variables': {
+                    'episodeOrSeasonOrSeriesId': asin,
+                },
+                'query': '''
+query getSeasons($sessionIdToken: String!, $deviceLocale: String, $episodeOrSeasonOrSeriesId: ID!, $clientId: String) {
+  getSeasons(
+    applicationContextInput: {deviceLocale: $deviceLocale, sessionIdToken: $sessionIdToken, clientId: $clientId}
+    episodeOrSeasonOrSeriesId: $episodeOrSeasonOrSeriesId
+  ) {
+    seasons {
+      seasonId
+    }
+  }
+}
+''',
+            },
+            note='Downloading series info')
+
+        for season in season_info['seasons']:
+            yield self.url_result(f'amazonminitv:{season["seasonId"]}', AmazonMiniTVSeasonIE, season['seasonId'])
```

Comment:
```suggestion
            yield self.url_result(f'amazonminitv:season:{season["seasonId"]}', AmazonMiniTVSeasonIE, season['seasonId'])
```

Series code is not working. This change will fix it